modules/tinymce/src/core/main/ts/keyboard/PageUpDownKeys.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { Cell } from '@ephox/katamari';
+import Editor from '../api/Editor';
+import { NodeChangeEvent } from '../api/EventTypes';
+import { EditorEvent } from '../api/PublicApi';
+import VK from '../api/util/VK';
+import * as InlineBoundariesNavigation from './InlineBoundariesNavigation';
+import * as MatchKeys from './MatchKeys';
+
+const executeKeyupAction = (editor: Editor, evt: KeyboardEvent) => {
+  MatchKeys.execute([
+    { keyCode: VK.PAGE_UP, action: MatchKeys.action(InlineBoundariesNavigation.moveOutInlineBoundaries, editor, false) },
+    { keyCode: VK.PAGE_DOWN, action: MatchKeys.action(InlineBoundariesNavigation.moveOutInlineBoundaries, editor, true) }
+  ], evt);
+};
+
+const stopImmediatePropagation = (e: EditorEvent<NodeChangeEvent>) => e.stopImmediatePropagation();
+
+const isPageUpDown = (evt: EditorEvent<KeyboardEvent>) => {
+  return evt.keyCode === VK.PAGE_UP || evt.keyCode === VK.PAGE_DOWN;
+};
```

Comment:
```suggestion
const isPageUpDown = (evt: EditorEvent<KeyboardEvent>) =>
  evt.keyCode === VK.PAGE_UP || evt.keyCode === VK.PAGE_DOWN;
```
We prefer shorthand arrow functions where possible generally. It's only if doing that makes it harder to read that we use the longhand format.

Reply:
updated